driver-core/src/test/resources/connection-string/valid-warnings.json

```diff
@@ -63,6 +63,51 @@
       "options": {
         "wtimeoutms": 10
       }
+    },
+    {
+      "description": "Empty integer option values are ignored",
```

Comment:
This throws an exception.  I don't think it's worth fixing so just skipping the test in code.